test/unit_tests/test_layers.py

```diff
@@ -9,7 +9,7 @@ class TestLayers:
     Testing PyTorchFI.Core example client.
     """
 
-    def setup_class(self):
```

Comment:
This should stay as "setup_class" to follow the pytest construct. Otherwise, this test doesn't conform to the standard.